atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -327,6 +330,15 @@ def __init__(self, data, pulp_data, odcs_config, koji_tag=None, arches=None):
             self.flags = None
             if data.get(UNPUBLISHED_REPOS):
                 self.flags = [UNPUBLISHED_REPOS]
+        for arch in data.get('multilib_arches', []):
+            if arch in arches:
+                self.multilib_arches.append(arch)
+        if self.multilib_arches:
+            if data.get('multilib_method'):
+                self.multilib_method = data.get('multilib_method')
```

Comment:
We're using `multlib_method` vs `multilib_methods` (plural) inconsistently. Let's just stick with one, probably `multilib_method` because that's what odcs wants.

Reply:
Fixed.